script.js

```diff
@@ -37,3 +38,183 @@ open.addEventListener('click', () => {
+  const project = document.createElement('div');
+  project.className = 'previous-work';
+  project.innerHTML = `
+  <section class="project">
+    <img class="snapshot" src=${image} alt="project 1 snapshot">
+    <div class="project-properties">
+      <h2>${title}</h2>
+      <ul class="tech">
+      <li class="company-name">${dev1}</li>
+      <li class="dev2">${dev2}</li>
+      <li class="dev3">${dev3}</li>
+      <li class="dev4">${dev4}</li>
+      <li class="dev5">${dev5}</li>
+      </ul>
+      <p>${description[1]}</p>
+      <ul class="prog-languages">
+      <li class="lang">${tag1}</li>
+      <li class="lang"  style="min-width: fit-content;">${tag2}</li>
+      <li class="lang">${tag3}</li>
+      <li class="lang">${tag4}</li>
+      </ul>
+      <button type='button' class="show-project">See Project</button>
+    </div>
+  </section>`;
+
+  main.appendChild(project);
+}
+
+for (let i = 0; i < 4; i += 1) {
+  show(i);
+}
+
+// Add popup window
+const seeProject = document.querySelectorAll('.show-project');
+
+function display(index) {
+  const main = document.querySelector('main');
+  const {
+    title,
+    devs,
+    description,
+    tags,
+    image,
+    liveButton,
+    sourceButton,
+    live,
+    source,
+  } = projects[index];
+  const dev1 = devs[0];
+  const dev2 = devs[1];
+  const dev3 = devs[2];
+  const dev4 = devs[3];
+  const dev5 = devs[4];
+  const tag1 = tags[0];
+  const tag2 = tags[1];
+  const tag3 = tags[2];
+  const tag4 = tags[3];
+
+  const projectPopup = document.createElement('div');
+  projectPopup.className = 'project-popup';
+  projectPopup.innerHTML = `
+    <div class="desktop-window">
+    <div class="all-popup">
+        <div class="desktop-top-popup">
+          <h2>${title}</h2>
+          <button type='button' class="close-button">X</button>
+          </div>
+        <ul class="tech">
+        <li class="company-name">${dev1}</li>
+        <li class="dev2">${dev2}</li>
+        <li class="dev3">${dev3}</li>
+        <li class="dev4">${dev4}</li>
+        <li class="dev5">${dev5}</li>
+        </ul>
+        <img class="desktop-popup-snapshot" src=${image} alt="project 1 snapshot">
+        <div class="bottom-container">
+          <p class="p-popup">${description[0]}</p>
+          <div class="small-container">
+            <ul class="prog-languages">
+              <li class="lang">${tag1}</li>
+              <li class="lang"  style="min-width: fit-content;">${tag2}</li>
+              <li class="lang">${tag3}</li>
+              <li class="lang">${tag4}</li>
+            </ul>
+            <hr class="line">
+            <section class="buttons">
+            <button type='button' class="popup-button">
+              ${liveButton}
+              <img class="button-logo" src=${live}>
+            </button>
+            <button type='button' class="popup-button">
+              ${sourceButton}
+              <img class="button-logo" src=${source}>
```

Comment:
- [ ] [Optional] As the previous reviewer mentioned you should use ```<a>``` tag instead of ```<img>``` to create the ```source link``` and the ```live link```

Reply:
Hi @Ismailco
Thanks for your review.
**The requirements you made are marked as “[optional]“, yet the project was not approved.**